content/cybersecurity/concepts/cyber-attack/terms/cross-site-scripting/cross-site-scripting.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Cross-Site Scripting (XSS)'
+Description: 'Cross-site scripting is a common cyber attack where an attacker embeds malicious code on a trusted website in order to trick a user into executing it.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Browsers'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'path/fundamentals-of-cybersecurity'
+---
+
+**Cross-site scripting**, also known as XSS, is a type of injection attack where an attacker successfully tricks the user into executing malicious code, usually in the form of a browser side script, sent through a trusted website. XSS attacks are one of the most common cyber attacks because many websites use input from users.
+
+## Types of XSS Attacks
+
+There are 3 common types of cross-site scripting attacks:
+1. Stored/Persistent XSS: The attacker attempts to get data provided from injected scripts placed in the victim's web browser, or the website's web server or database.
+
+2. Reflected/Non-Persistent XSS: The attacker attempts to have the user click on a malicious link, masqueraded as a legitimate link, from the website's error message, search result, or phishing email.
+
+3. Document Object Model (DOM) Based XSS: With DOM-based XSS, the attacker attempts to exploit the victim's web browser by attacking the DOM, usually through the URL. This attack is also known as client-side XSS.
```

Comment:
```suggestion
There are three common types of cross-site scripting attacks:

1. Stored/Persistent XSS: The attacker attempts to get data provided from injected scripts placed in the victim's web browser, or the website's web server or database.
2. Reflected/Non-Persistent XSS: The attacker attempts to have the user click on a malicious link, masqueraded as a legitimate link, from the website's error message, search result, or phishing email.
3. Document Object Model (DOM) Based XSS: With DOM-based XSS, the attacker attempts to exploit the victim's web browser by attacking the DOM, usually through the URL. This attack is also known as client-side XSS.
```
Adjust spacing. Should be a blank line before bulleted list and no blank lines between items.